spec/lib/tacos_demo.py

```diff
@@ -213,25 +213,15 @@ def parse_comments(
 def parse_comment(
     job_filter: str | None, slices_subpath: Path, comment: str
 ) -> Generator[tuple[str, Slice, str]]:
-    remainder = comment
-    while True:
-        comment, tag_start, remainder = remainder.partition(COMMENT_TAG)
-        if not tag_start:
-            return
-
-        tag, tag_end, remainder = remainder.partition(COMMENT_TAG_END)
-
-        tag = Parse(tag)
-        job = tag.before.first("(")  # )
+    lines = comment.splitlines(keepends=True)
+    points = [i for i, l in enumerate(lines) if COMMENT_TAG in l]
+    points.append(len(lines))
+    chunks = ["".join(lines[i:j]) for i, j in pairwise(points)]
+    for chunk in chunks:
+        before, _tag_start, after = chunk.partition(COMMENT_TAG)
+        job, _tag_end, after = after.partition(COMMENT_TAG_END)
         if job_filter is not None and job != job_filter:
             continue  # this is not the job_filter you're looking for
-
-        comment = "".join((comment, tag_start, tag, tag_end))
-
-        parsed = tag.between("(", ")")
-        if parsed == tag:
-            slice = Slice(".")
-        else:
-            slice = Slice(parsed)
-            slice = slice.relative_to(slices_subpath)
-        yield job, slice, comment
+        slice = Slice(before[before.index("terraform/") : -1])
```

Comment:
does this mean `terraform` has to be part of the path now?

Reply:
yes, which I think is always the case for the tests. this was sort of desperation, as there's not a reliable delimiter on the left side of the slice. may need to think on this part more.

Reply:
There's reliably a space on the left side.

Reply:
seems like there's not when it's a link in markdown?